lib/importer/views/plugin_config.html

```diff
@@ -93,7 +93,40 @@ <h2 class="govuk-heading-l">Column settings</h2>
       </div>
 
     </div>
-  </div>
+      <form action="config" method="post">
+
+    <div class="govuk-grid-column-full govuk-!-padding-top-6" >
+        <h2 class="govuk-heading-l"></h2>
+
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset" aria-describedby="waste-hint">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+      <h1 class="govuk-fieldset__heading">
+        Report usage data
+      </h1>
+    </legend>
+
+    <p class="govuk-body">
+      Allow Register Dynamics to collect usage data on your use of the Data Upload Design Kit.
+      When enabled, the following events will be reported with no further data:
```

Comment:
Can we include that we will collect "an anonymous ID specific to this prototype"? As it would be good to collect the client ID.

Reply:
Fixed in 67e7459